Marlin/ultralcd_impl_DOGM.h

```diff
@@ -410,6 +410,7 @@ FORCE_INLINE void _draw_axis_label(const AxisEnum axis, const char* const pstr,
 
 inline void lcd_implementation_status_message() {
   #if ENABLED(STATUS_MESSAGE_SCROLLING)
+    const bool blink = lcd_blink();
```

Comment:
This is patched in another PR still pending. In that patch the function takes `blink` as an input argument.